designer/server/src/test/scala/pl/touk/nussknacker/test/base/it/NuResourcesTest.scala

```diff
@@ -186,7 +186,7 @@ trait NuResourcesTest
   protected val typeToConfig: ProcessingTypeDataProvider[ProcessingTypeData, CombinedProcessingTypeData] = {
     val designerConfig = DesignerConfig.from(testConfig)
     TestProcessingTypeDataProviderFactory.fromState(
-      new ProcessingTypesConfigBasedProcessingTypeDataLoader(() => IO.pure(designerConfig.processingTypeConfigs))
+      new ProcessingTypeDataLoader(() => IO.pure(designerConfig.processingTypeConfigs))
```

Comment:
It doesn't change much here (because it's a test code), but when you have a not-evaluated-yet value, it's better to use `IO.delay` instead of `IO.pure`

Reply:
I know that, in next changes, I removed `IO` from `ProcessingTypeDataLoader` parameter because it wasn't used only to do `_.map(...)` on the beginning which was redundant complication.